plugins/woo-ai/includes/class-woo-ai-settings.php

```diff
@@ -168,74 +96,58 @@ public function add_woo_ai_settings( $settings = array(), $current_section = nul
 	}
 
 	/**
-	 * Return array describing new settings.
-	 *
-	 * @return array Describing new AI settings.
+	 * Add Woo AI Settings to section.
 	 */
 	public function get_woo_ai_settings() {
+		$settings_ai = array();
 
-		// These are made available in the DOM and actually toggled client-side.
-		$tone_desc = array(
-			'informal'     => __( 'Relaxed and friendly, as if you were having a conversation with a friend.', 'woocommerce' ),
-			'humorous'     => __( 'Engage customers with a light-hearted and fun communication style.', 'woocommerce' ),
-			'neutral'      => __( 'A balanced and impartial tone that uses casual expressions without slang.', 'woocommerce' ),
-			'youthful'     => __( 'Bring energy and enthusiasm to your store with a friendly and cheeky tone.', 'woocommerce' ),
-			'formal'       => __( 'Direct yet respectful, a formal tone sounds serious and professional.', 'woocommerce' ),
-			'motivational' => __( 'Passionate and inspiring, engage users with this upbeat tone.', 'woocommerce' ),
+		$settings_ai[] = array(
+			'id'    => self::WOO_AI_TITLE_OPTION_KEY,
+			'title' => __( 'Artificial Intelligence', 'woocommerce' ),
+			'desc'  => __( "Save time by automating mundane parts of store management. This information will make AI-generated content, visuals, and settings more aligned with your store's goals and identity.", 'woocommerce' ),
+			'type'  => 'title',
 		);
 
-		$markup_str = array_reduce(
-			array_keys( $tone_desc ),
-			function( $acc, $key ) use ( $tone_desc ) {
-				$value = $tone_desc[ $key ];
-				return $acc . '<span class="woo-ai-settings-tone-option" data-option="' . htmlspecialchars( $key ) . '">' . htmlspecialchars( $value ) . '</span>';
-			},
-			''
+		$settings_ai[] = array(
+			'id'      => self::WOO_AI_ENABLED_OPTION_KEY,
+			'title'   => __( 'Enable AI', 'woocommerce' ),
+			'desc'    => __( 'Enable AI features in your store', 'woocommerce' ),
+			'type'    => 'checkbox',
+			'default' => 'yes',
 		);
 
-		return array(
-			array(
-				'id'    => 'woo_ai_title',
-				'type'  => 'title',
-				'title' => __( 'Artificial Intelligence', 'woocommerce' ),
-				'desc'  => __( "Save time by automating mundane parts of store management. This information will make AI-generated content, visuals, and setting more aligned with your store's goals and identity.", 'woocommerce' ),
-			),
-
-			array(
-				'title'   => __( 'Enable AI', 'woocommerce' ),
-				'desc'    => __( 'Enable AI features in your store', 'woocommerce' ),
-				'id'      => 'woo_ai_enable_checkbox',
-				'default' => 'yes',
-				'type'    => 'checkbox',
+		$settings_ai[] = array(
+			'id'      => self::TONE_OF_VOICE_OPTION_KEY,
+			'name'    => __( 'Tone of voice', 'woocommerce' ),
+			'desc'    => __( 'Select the tone of voice for the AI', 'woocommerce' ),
+			'type'    => 'select',
+			'options' => array(
+				'informal'     => __( 'Relaxed and friendly.', 'woocommerce' ),
```

Comment:
This makes more sense than just including one word. I'd be interested to see how the model behaves if we included these descriptions in the prompt instead of the single word describing the tone. I think these will give the model more info on what we expect in the result.

Also tagging @jarekmorawski here to double check the wording for these options.